pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -174,7 +174,7 @@ func newCertRotationController(
 			Namespace:              operatorclient.OperatorNamespace,
 			Name:                   "kube-apiserver-to-kubelet-signer",
 			Validity:               1 * 365 * defaultRotationDay, // this comes from the installer
-			Refresh:                8 * 365 * defaultRotationDay, // this means we effectively do not rotate
+			Refresh:                292 * defaultRotationDay,
```

Comment:
we refresh others in  halves

Reply:
I am not sure how gracefully kubelet handles it, so I was just explicitly codifying existing behaviour so this PR is noop. If you sign up for 1/2 I can adjust that.